runtime/list_test.go

```diff
@@ -392,22 +392,20 @@ func TestListSetItem(t *testing.T) {
 }
 
 func TestListSort(t *testing.T) {
-	fun := newBuiltinFunction("TestListSetItem", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
-		if raised := checkFunctionArgs(f, "TestListSetItem", args, ListType); raised != nil {
-			return nil, raised
-		}
-		l := toListUnsafe(args[0])
-		if raised := l.Sort(f); raised != nil {
+	fun := newBuiltinFunction("TestListSort", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+		if _, raised := listSort(f, args, KWArgs{}); raised != nil {
```

Comment:
I prefer to get the list.sort attribute and then Call(). Using listSort() directly is kind of like a back door. You can fetch it once above `fun` and then use it on each test:

```
sort := mustNotRaise(GetAttr(NewRootFrame(), ListType.ToObject(), NewStr("sort"), nil))
fun := ... {
  if _, raised := sort.Call(f, args, nil); raised != nil {
  ...
```